### ISSUE TITLE
.net standard 2.1 not compatable with NETFrameworks

### ISSUE THREAD

**Issue by LeoJHarris** [thudugala/Plugin.LocalNotification#80]
Upgrading to latest on this library is causing issues with my UITest project which uses NETFramework 4.6.1 . 

netstandard 2.1 is not compatable with any .NETFramework: https://docs.microsoft.com/en-us/dotnet/standard/net-standard

Error is below failing my build:

 sampleApp targets 'netstandard2.1'. It cannot be referenced by a project that targets '.NETFramework,Version=v4.6.1'

Im guessing latest version on this plugin prevents referencing projects with NETFramework

I try a work around by dropping to version 4.1.4 but I get issues with linking which I think is the linker stripping out some classes maybe? I dont want to set linking to none but if this is a linker issue any ideas how to fix this? Again guessing I need a line to the proguard.cfg? If so some suggestions be helpful:

```csharp
Mono.Linker.MarkException: Error processing method: 'System.Void XeniaGuestAppii.Droid.Resource::UpdateIdValues()' in assembly: 'XeniaGuestAppii.Android.dll' ---> Mono.Cecil.ResolutionException: Failed to resolve System.Int32 Plugin.LocalNotification.Resource/Attribute::coordinatorLayoutStyle
```

**Comment by thudugala** [comment]
@LeoJHarris  Can you please attach a sample project

**Comment by LeoJHarris** [comment]
@thudugala Yes I can get try get a sample, setting off the linker works for the project to get it to succees the build. Any immediate ideas?

Although I'm pretty sure adding a certain line to the proguard.cfg will fix the issue, Ill investigate around that first since it might be an easy fix otherwise.

**Comment by thudugala** [comment]
also look into #72

**Comment by LeoJHarris** [comment]
@thudugala is there a reason for using .net standard 2.1? Can it not be using 2.0 in latest? Going forward it will have issues with project libraries such as any unit testing projects.

**Comment by thudugala** [comment]
Version 5 and above is targeting Android 10 (this cannot be changed because of android dependency).
Can you please verify that changing to .net standard 2.0 will help you ? I think  .NET Framework does not support Android 10.

**Comment by LeoJHarris** [comment]
Ill send a sample project as i am unable to get the build working unfortunately. Ill try get something this week.

**Comment by LeoJHarris** [comment]
@thudugala I migrated my Uni testing project to .net core so I could use latest version of your package and no issue now.

**Comment by LeoJHarris** [comment]
> Can you please verify that changing to .net standard 2.0 will help you ? I think .NET Framework does not support Android 10.

If your just unit testing a .net standard project i.e. the shared code then it shouldnt matter about the android 10 support with .NET Framework?

**Comment by thudugala** [comment]
I think Android 10 is only a issue for UITest.

**Comment by LeoJHarris** [comment]
I actually raised a question on the forums around .NET Framework vs .Net Core. Do you happen to use .Net Core over .NetFramework when unit testing?
